contracts/Governance.sol

```diff
@@ -0,0 +1,925 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/utils/cryptography/ECDSA.sol';
+import '@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol';
+import '@openzeppelin/contracts/utils/introspection/ERC165.sol';
+import '@openzeppelin/contracts/utils/Counters.sol';
+import '@openzeppelin/contracts/utils/Address.sol';
+import '@openzeppelin/contracts/utils/Context.sol';
+import '@openzeppelin/contracts/utils/Timers.sol';
+import '@openzeppelin/contracts/utils/math/SafeCast.sol';
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/governance/TimelockController.sol';
+
+// BitDAO token contract interface.
+interface IERC20Votes is IERC20 {
+    function getCurrentVotes(address account) external returns (uint256);
+}
+
+// Governance contract.
+contract Governance is Context, ERC165, EIP712 {
```

Comment:
For ERC165 do we need an implementation of `supportsInterface` that covers all the interfaces the `Governance` implements?

Reply:
No, at least not for now, its just for override